.gitignore

```diff
@@ -3,4 +3,6 @@
 /.tmp
 /dist
 
-.DS_Store
\ No newline at end of file
+.DS_Store
+*.sublime-project
```

Comment:
Should this be in `~/.gitignore`? (probably same with `~/.DS_Store` now that I think about it)